images/macos/toolsets/Readme.md

```diff
@@ -6,8 +6,18 @@
   - `link` property points to the place where Xcode will be located on image. `/Applications/Xcode_<link>.app`  
   - `version` points to Xcode version that will be downloaded and installed
   - `symlinks` describes the list of aliases where symlinks will be created to
-  - `install_runtimes` is an array or boolean function to control over the related simulator runtimes, set of possible values: [ `iOS`, `watchOS`, `visionOS`, `tvOS` ], use `true` if you want to install all runtimes, use `false` if you want to skip runtimes installation
   - `sha256` used to check integrity of the Xcode installer file
+  - `install_runtimes` – controls the installation of simulator runtimes:
+    - `"all"` – installs all default runtimes.
+    - `false` – skips runtime installation.
```

Comment:
nit: Consider using same kinds of words like `all`/`none` or `true`/`false`. It is also confusing that `"all"` is quoted but `false` is not (I understand why, but still...)